### PR TITLE
Removed first click free

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: demo/main.js [hunk 1]
```diff
@@ -23,8 +23,7 @@ app.set('view engine', 'html');
 app.use(express.static(path.resolve(__dirname, 'public')));
 app.use(function(req, res, next){
 	res.locals.flags = {
-		barrier : true,
-		firstClickFree : false
```
Comment (aintgoin2goa) [inline demo/main.js]: can we keep this flag please?  The barriers middleware uses this so if useful for testing

Comment (adamsilver) [inline demo/main.js]: Do you mean to keep firstClickFree flag or barrier flag? The barrier flag is below...

Comment (aintgoin2goa) [inline demo/main.js]: sorry, the flrstClickFree flag please - the middleware checks this one

**Comment by aintgoin2goa** [comment]
:+1:

**Pull request merged**